frontend/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
what is this? does the old frontend work?

Reply:
this is a part of the create-react app